torch/_expanded_weights/expanded_weights_utils.py

```diff
@@ -0,0 +1,116 @@
+import torch
+from .expanded_weights_impl import ExpandedWeight
+
```

Comment:
Nothing in this file gets used anymore now that we've deleted the fallback, right? Maybe adding a single simple per-sample-grad rule to this PR (like the one for linear) would help demonstrate how these are used, especially for testing

Reply:
(Or just moving this file to the next PR or somewhere else)